ssl/d1_lib.c

```diff
@@ -1088,3 +1091,39 @@ unsigned int dtls1_min_mtu(SSL *s)
 {
     return dtls1_link_min_mtu() - BIO_dgram_get_mtu_overhead(SSL_get_wbio(s));
 }
+
+static unsigned int dtls1_get_data_mtu(SSL *s)
+{
+	unsigned int mac_overhead, int_overhead, blocksize, ext_overhead;
+	const SSL_CIPHER *ciph = SSL_get_current_cipher(s);
+	unsigned mtu = s->d1->mtu;
```

Comment:
I am currently working on a size_t-ify libssl branch which converts s->d1->mtu to size_t. Should we anticipate that branch and just make it a size_t here and then (if we make this a proper function rather than a ctrl):

````
size_t DTLS_get_data_mtu(SSL *s);
````

I know that is inconsistent with the other mtu getting functions....but still...